### PR TITLE
Initial setup for VR&E chapter 36

### PR DESCRIPTION
**Pull request by jbalboni**
Not really anything to see. This sets up an empty config and puts the other boilerplate in place.
  
screen shot 2018-01-08 at 1 14 35 pm

### CODE REVIEW

**Review by cehsu — APPROVED** [review]
LGTM. Had a few minor questions, also unrelated to these changes, it looks like "36" was accidentally replaced with "31" in content/pages/employment/vocational-rehab-and-employment.md?
Also, should src/js/vre/chapter36/config/.form.js.swm be included?

File: src/js/vre/chapter36/components/IntroductionPage.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    focusElement('.va-nav-breadcrumbs-list');
+  }
+  goForward = () => {
+    this.props.router.push(this.props.route.pageList[1].path);
+  }
+  render() {
+    return (
+      <div className="schemaform-intro">
+        <FormTitle title="Apply for VR&E"/>
+        <p>Equal to form chapter 36</p>
```
[the remark this answers is not in the document]
Comment (cehsu) [reply src/js/vre/chapter36/components/IntroductionPage.jsx]: Also should more information be provided? For example, in the health care application we provide the form number and a description in parentheses:
Equal to VA Form 10-10EZ (Application for Health Benefits).

[the remark this answers is not in the document]
Comment (jbalboni) [reply src/js/vre/chapter36/components/IntroductionPage.jsx]: It should, but I was just putting placeholder content in here. There's an issue to update the Introduction page where we would want to update all this text.

File: src/js/vre/chapter36/components/IntroductionPage.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        <SaveInProgressIntro
+          prefillEnabled={this.props.route.formConfig.prefillEnabled}
+          pageList={this.props.route.pageList}
+          startText="Start the Chapter 36 application"
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}>
+          Please complete the Chapter 36 form to apply for benefits
```
[the remark this answers is not in the document]
Comment (jbalboni) [reply src/js/vre/chapter36/components/IntroductionPage.jsx]: I think it's "vocational counseling," but like I said above, I'm leaving most of this to be updated in the intro page ticket.

**Pull request merged**